recipes/airss/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "airss" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}-pkg
+  version: {{ version }}
+
+source:
+  - url: https://www.mtg.msm.cam.ac.uk/files/airss-{{ version }}.tgz 
+    sha256: b54c61b2988b3e6ee6d01e8b73fc60d76f36ad47eeb459d1a0358e486195cc13
+    patches:
+      - patches/makefile.patch
+  - url: https://github.com/spglib/spglib/archive/refs/tags/v1.16.5.zip
+    sha256: fa8f6eb0dc7bb0410f517038e3a165ce74549d19c41a8bdbb34864c7141304a1
+    folder: external/spglib/spglib-1.16.5
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+outputs:
+  - name: {{ name }}
+    script: build_airss.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('fortran') }}
+        - make
+        - patch
+      host:
+        - spglib 1.16.*
+        - libblas
+        - liblapack
+        - perl
```

Comment:
Please address multiple overdepending/underlinking warnings

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=661915&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2524

Reply:
Not providing `libblas` leads to incorrect linking of Lapack/BLAS:
````
  ERROR (airss,libexec/airss/cabal): Needed DSO lib/libblis.so.4 found in ['blis']
  ERROR (airss,libexec/airss/cabal): .. but ['blis'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
````
Keeping `libblas` with a comment in req/host.

Reply:
Then use `ignore_run_exports` to prevent the package from depending on an unused libblas.